clients/drcachesim/analyzer_multi.cpp

```diff
@@ -450,7 +450,8 @@ analyzer_multi_tmpl_t<RecordType, ReaderType>::analyzer_multi_tmpl_t()
                 nullptr, op_verbose.get_value(), op_jobs.get_value(),
```

Comment:
> We have observed sporadic decoding issues raised by libipt which are not yet root-caused; these may be due to our own usage of perf_event_open to record PT data, or our use of libipt may not be robust enough to handle these errors.

I thought a primary cause was our kcore dump not being large enough to include some jitted code?

Reply:
I think I fixed that by increasing the range around the BPF JIT code that's included in the dump. What's left are:

- "expected tracing enabled event" due to insufficient kernel trace buffer size. But if I increase it any further it causes OOMs.
- "trace stream does not match query"

As in the PR description, these seem to be for futex and switchto.